hardware/doc/SNAP-Registers.md

```diff
@@ -193,6 +193,7 @@ Address: 0x0000030
   15..9  RO: Reserved
       8  RO: NVMe enabled
    7..0  RO: Card type:
+             0x11 : RCXVUP
```

Comment:
The corresponding change is missing in the slave context register section (lines 505 ff.)

Reply:
corrected.